src/haddock/modules/scoring/sasascore/sasascore.py

```diff
@@ -0,0 +1,164 @@
+"""Accessibility scoring calculations."""
+from haddock import log
+from pathlib import Path
+import pandas as pd
+from haddock.core.typing import FilePath
+from haddock.clis.restraints.calc_accessibility import (
```

Comment:
...
Should work...
Until removed from the code ...
...

Reply:
Exactly, these functions should be moved here - but we can do that then deleting the restraints CLIs